Update installer

flokuep opened the pull request:
This updates our installer, closes #289

- Updates installer to our current files
- Updates advanced installer version
- Separate x86 and x46 installer to decrease file size
- Automatically install .NET 4.0 and vc2010 runtime if not installed
- Ask during uninstallation if data folders should be deleted or not
- Fixes some issues with "Installer" build target
- New batch script for easier building installer

For testing:
- x86: https://cloud.floriankuepper.de/index.php/s/QxOfR4gUKdviYGc
- x64: https://cloud.floriankuepper.de/index.php/s/5DauSMoSNhtVyM0

lukeIam commented:
From my dependency tests:
I uninstalled VC2012 an it's still working (if the check in `CProgrammHelper.cs` is disabled)
at least I could start Vocaluxe and play a song with video - no audio, but that was a vm problem (no audio device there)

So system dependencies are:
- VC2010
- DirectX 9c
- .NET 4.0

-> Summary: everything als expected...

lukeIam commented on Vocaluxe/CProgrammHelper.cs:
```diff
@@ -116,13 +116,20 @@ private static void _EnsureDataFolderExists()
         {
 #if WIN
-            if (!_IsVC2012Installed())
+            if (!_IsVC2010Installed())
```
@flokuep 
I removed the VC2012 check (seems not to be needed anymore) 
I added VC2010 check (you got DirectX missing error, if DirectX was installed but VC2010 not)

flokuep replied:
Okay. Thanks.

BenKmann commented:
I tried the x64 Setup and it worked fine. Installed correctly and game starts.
I don't know if it has to, but the setup didn't create a "config.xml" File. That one is missing.
The Game also didn't create one (in the Readme it is still referenced).

flokuep commented:
> Or am I overlooking something?

@lukeIam No, you're right. I've changed it.

> I tried the x64 Setup and it worked fine. Installed correctly and game starts. I don't know if it has to, but the setup didn't create a "config.xml" File. That one is missing. The Game also didn't create one (in the Readme it is still referenced).

@BenKmann Thanks for your feedback. You can find config.xml in Documents/Vocaluxe. We need to put it there, because maybe we haven't write permission on install folder during running vocaluxe. I've updated readme about this, just need to merge it ;)
